_data/authors.yml

```diff
@@ -190,6 +190,12 @@
   description: is a JavaScript Engineer, Node.js lover, and Elixir alchemist.
   image: emueller.jpg
 
+- id: eprohn
+  name: Elisa
+  full_name: Elisa Prohn
+  description: is a passionate HR Manager, who loves the variety and challenges in her daily working life. She believes in building employee engagement and enjoys working with our collaborative colleagues.
```

Comment:
We need to shorten this a bit as it is much longer than the two other ones.

Suggestion: is a passionate HR Manager, who loves the variety of challenges in her daily working life. She strongly believes in building employee engagement.